index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Page</title>
+</head>
+<body>
+	
+  <header class="about-me">
+		<div class="menu-bar">
+      <a href="#">my logo</a>
+      <img src="img/Union.svg" alt="Hamburger">
+		</div>
+    <section class="about">
+      <div class="intro">
+        <h2>I’m Victor Ozoede Glad to see you!</h2>
```

Comment:
### OPTIONAL
- [ ] You can change `h2` tag with `h1` tag. It is good for seo. [What is the H1 Tag and Why it is Important for SEO ](https://www.reliablesoft.net/h1-tag/#:~:text=The%20HTML%20tag%20is%20the%20first%20header%20tag%20visible,rest%20of%20the%20page%20tags.)